index.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express")
+const { GracefulShutdownServer } = require("medusa-core-utils")
+
+const loaders = require("@medusajs/medusa/dist/loaders/index").default
```

Comment:
thought: In my experiment, I decided to simply surface the `medusa start` command. The reasoning was around maintainability since both `medusa start` (official way from our docs) and `index.js` would follow the same code path. That being said, I didn't explore new opportunities from a different approach. Againg, just a thought.
```javascript
const start = require("@medusajs/medusa/dist/commands/start").default;
const run = async () => {
  const port = process.env.PORT || 9000
  const directory = process.cwd();
  await start({ port, directory });
}
void (async () => {
  await run()
})()
```

Reply:
The idea of the entry point is to also have the ability to access express directly, which is not the case by using the start command here, though, i am fine with both approaches but I thought that mine would explicit what is today implicit and allow easier customisation from the end user as it is often the needs as we have seen with lots of companies. Wdyt?